test_suite/generic/test_generic.py

```diff
@@ -151,6 +152,27 @@ def test_inittab_and_systemd(self, host):
                     assert 'si::sysinit:/etc/rc.d/rc.sysinit' in host.check_output("grep '^si:' /etc/inittab"), \
                         'Unexpected default inittab "id"'
 
+    def test_release_version(self, host, instance_data):
+        '''
```

Comment:
Replace single quotes by double quotes for multi-line strings and docstrings. I think flake8 also highlights this.